src/Tribe/Integrations/ACF/resources/tribe-admin-acf-compat.js

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
I think you forgot to upload the minified version of this.

Reply:
there's no minified version of the script; i don't think we'd get value out of having one, and since tribe_asset is not going to be used, the overhead to correctly load the min or non-min file is also not necessary